src/backend/index.js

```diff
@@ -31,6 +32,7 @@ export function initBackend (_bridge) {
       initVuexBackend(hook, bridge)
     })
   }
+  initEventsBackend(rootInstances[0], bridge, instanceMap, getInstanceName)
```

Comment:
@yyx990803 I don't know if this is the right approach!? I grab the first root instance of the page and patch the prototype there.